### ISSUE TITLE
Move related projects, both issues and pull requests from espes/xqemu, JayFoxRox/xqemu and utils to xqemu org

### ISSUE THREAD

**Issue by timofonic** [JayFoxRox/xqemu#28]
Hello.
  @espes @JayFoxRox 
I consider being essential to migrate all the project content to the new official organization. I think all the issues and merge requests (even closed ones for reference) should be migrated to the official repo ASAP.

We can ask for assistance. I'm a newbie, but I can help at formatting texts in AsciiDoc or MarkDown.

What's the showstopper?

Kind regards.

**Comment by JayFoxRox** [comment]
This is my work repo, all PRs / issues here affect only my WIP stuff. It's not a seperate project / fork with a different goal.
Those which are closed *have* been moved to the master repo. Those still opened *will* be moved to the master repo.

espes repo on the other hand was the repo where all the work was PR'd to so far (as espes started the project = it was the first repo in existence).
My hope was to have xqemu/xqemu as a central workplace instead, but I think espes did not like that approach and the migration felt very half-assed.
Lately, I also did not enjoy the review process and how the project is structured (public-source / private project, not open-source). PR 92 also took far too long for my taste and it was an important issue to me. I think doing PR 92 far earlier would have saved me days / weeks of work, and I also thought it was "odd" that a contributor even had to bring it up.

Long story short: I was / am frustrated with the whole situation and I'm currently considering to move away from xqemu to work on other / another projects. I'll still be contributing to the wiki and nxdk / nxdk-rdt.
I'll have to make up my mind about xqemu itself.
